src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -46,7 +46,17 @@
     </td>
     <td class="file-status">
       <div>
-        {{ distro_tag(version.channel)}} {{ version.file.get_status_display() }}
+        {{ distro_tag(version.channel)}}
+        <span class="file-status-text">
+          {% if version.file.status == amo.STATUS_DISABLED and version.is_blocked %}
+            {# Slight optimization: don't try getting blocked status for
+               non-disabled versions - this avoids checking if for latest/current
+               versions, which are fetched without a select_related('blockversion') #}
+            {{ version.blockversion.get_user_facing_block_type_display() }}
+          {% else %}
+            {{ version.file.get_status_display() }}
```

Comment:
I see we did this already, but I wonder why we weren't using `version.get_review_status_display` here?

Reply:
Good question. I think `get_review_status_display` was meant for reviewers only. It can return `Disabled by Developer` which is technically more correct that what we do now (atm in devhub if the developer disables the version, we say "Disabled by Mozilla") but at the same time a bit weird to display to a developer maybe ?

Reply:
We have a thread with UX about the various statuses and what we show to developers, lets wait to see how that shakes out as it will probably require more changes/its own issue.